spec/rubocop/cop/rspec/single_argument_message_chain_spec.rb

```diff
@@ -47,6 +47,49 @@
         end
       RUBY
     end
+
+    context 'with single-key hash argument' do
+      it 'reports an offence' do
+        expect_violation(<<-RUBY)
+          before do
+            allow(foo).to receive_message_chain(bar: 42)
+                          ^^^^^^^^^^^^^^^^^^^^^ Use `receive` instead of calling `receive_message_chain` with a single argument.
+          end
+        RUBY
+      end
+
+      include_examples(
+        'autocorrect',
+        'before { allow(foo).to receive_message_chain(bar: 42) }',
+        'before { allow(foo).to receive(:bar) { 42 } }'
+      )
+
+      include_examples(
+        'autocorrect',
+        'before { allow(foo).to receive_message_chain("foo bar" => 42) }',
+        'before { allow(foo).to receive("foo bar") { 42 } }'
+      )
+    end
+
+    context 'with multiple keys hash argument' do
+      it "doesn't report an offence" do
+        expect_no_violations(<<-RUBY)
+          before do
+            allow(foo).to receive_message_chain(bar: 42, baz: 42)
+          end
+        RUBY
+      end
+    end
+
+    context 'with empty hash' do
+      it "doesn't report an offence" do
+        expect_no_violations(<<-RUBY)
+          before do
+            allow(foo).to receive_message_chain({})
```

Comment:
I’m not sure what `receive_message_chain({})` does. Intuitively, I’d think it doesn’t do anything.